packages/aws-rfdk/lib/core/lib/pad-efs-storage.ts

```diff
@@ -0,0 +1,316 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path';
+import {
+  ISecurityGroup,
+  IVpc,
+  SecurityGroup,
+  SubnetSelection,
+  SubnetType,
+} from '@aws-cdk/aws-ec2';
+import {
+  IAccessPoint,
+} from '@aws-cdk/aws-efs';
+import {
+  Code,
+  FileSystem as LambdaFilesystem,
+  Function as LambdaFunction,
+  Runtime,
+} from '@aws-cdk/aws-lambda';
+import {
+  RetentionDays,
+} from '@aws-cdk/aws-logs';
+import {
+  Choice,
+  Condition,
+  Fail,
+  InputType,
+  StateMachine,
+  Succeed,
+}from '@aws-cdk/aws-stepfunctions';
+import {
+  LambdaInvoke,
+} from '@aws-cdk/aws-stepfunctions-tasks';
+import {
+  Construct,
+  Duration,
+  Stack,
+} from '@aws-cdk/core';
+import {
+  AwsSdkCall,
+  AwsCustomResource,
+  AwsCustomResourcePolicy,
+  PhysicalResourceId,
+} from '@aws-cdk/custom-resources';
+import {
+  tagConstruct,
+} from './runtime-info';
+
+/**
+ * Input properties for PadEfsStorage.
+ */
+export interface PadEfsStorageProps {
+  /**
+   * VPC in which the given access point is deployed.
+   */
+  readonly vpc: IVpc;
+
+  /**
+   * PadEfsStorage deploys AWS Lambda Functions that need to contact your Amazon EFS mount target(s).
+   * To do this, AWS Lambda creates network interfaces in these given subnets in your VPC.
+   * These can be any subnet(s) in your VPC that can route traffic to the EFS mount target(s).
+   *
+   * @default All private subnets
+   */
+  readonly vpcSubnets?: SubnetSelection;
+
+  /**
+   * Amazon EFS Access Point into which the filesystem padding files will be added. Files will
+   * be added/removed from the root directory of the given access point.
+   * We strongly recommend that you provide an access point that is for a dedicated padding-files
+   * directory in your EFS filesystem, rather than the root directory or some other in-use directory
+   * of the filesystem.
+   */
+  readonly accessPoint: IAccessPoint;
+  /**
+   * Security group for the AWS Lambdas created by this construct.
+   *
+   * @default Security group with full egress and no ingress will be automaticaally created for each Lambda.
```

Comment:
```suggestion
   * @default Security group no egress or ingress will be automatically created for each Lambda.
```

Looks like we are setting `allowAllOutbound: false` in the default security group below

Reply:
Heh. Yup. Right you are!